smt/utils/design_space.py

```diff
@@ -299,7 +299,9 @@ def _decode_dv(x_encoded: np.ndarray, i_dv_decode):
         ]
         return decoded_des_vectors[0] if is_1d else decoded_des_vectors
 
-    def sample_valid_x(self, n: int, unfolded=False) -> Tuple[np.ndarray, np.ndarray]:
+    def sample_valid_x(
```

Comment:
Why not having `random_state` and `new_sampler` as arguments with default values? `kwargs` is less explicit and there is no documentation for the new options here.

Reply:
It was designed for EGO but still is an option and a feature of a `DesignSpace` method. Why not `random_state=None, use_new_sampler=False` as defaults?

Finally, I am not  sure to understand why `use_new_sampler` is required by EGO. Why using the same LHS sampler would be bad?